tests/functional/aws-node-sdk/lib/utility/bucket-util.js

```diff
@@ -7,12 +7,16 @@ class BucketUtility {
     constructor(profile = 'default', config = {}) {
         const s3Config = getConfig(profile, config);
 
-        this.s3 = Promise.promisifyAll(new S3(s3Config), { suffix: 'Promise' });
+        this.s3 = new S3(s3Config);
+        this.s3.config.setPromisesDependency(bluebird);
```

Comment:
we are using some features of bluebird library, we may remove it maybe, but we need to check if we have an alternative, but it's a good idea to get rid of it if possible, i would rather see this is a different ticket (targetting 7.9 maybe since it's not really for LTS purpose :))

Reply:
Indeed, we no longer need it. We should migrate away in a separate PR.